### PR TITLE
Install or upgrade to BlueZ 5.44, closes #20

### LINKED ISSUE #20: Update playbooks to install/test Bluez 5.44
Readme in Linux SDK repo should be updated as well if 5.44 still works as expected

### PR DESCRIPTION
**Pull request by grunskis**
Install BlueZ 5.44 or upgrade if you have an older version already installed.

### CODE REVIEW

File: deployment/roles/ubuntu-bluez/tasks/from_src.yml [hunk 1]
```diff
@@ -73,11 +73,19 @@
...
+
+- name: extract bluez version number from dict
+  set_fact:
+    custom_bluez_version: "{{ custom_bluez_version.stdout|default('') }}"
+
 - name: install sources
   command: make install
   args:
     chdir: "{{build_dir}}"
-    creates: /usr/local/bin/bluetoothctl
```
Comment (larsblumberg) [inline deployment/roles/ubuntu-bluez/tasks/from_src.yml]: Why is this line removed? Is it not necessary to check whether `/usr/local/bin/bluetoothctl` was created?

Comment (grunskis) [inline deployment/roles/ubuntu-bluez/tasks/from_src.yml]: It would skip the installation step if the file exists. In case of upgrade that's not what we want.

Comment (larsblumberg) [inline deployment/roles/ubuntu-bluez/tasks/from_src.yml]: Clever!
